docs/guides/testing.md

```diff
@@ -10,6 +10,8 @@ In this section, let's test with Cloudflare Workers and Jest.
 All you do is create a Request and pass it to the Hono application to validate the Response.
 And, you can use `app.request` the useful method.
 
+> For a typed test client see the [testing helper](/docs/helpers/testing).
```

Comment:
How about using the `Tip` box?

https://vitepress.dev/guide/markdown#default-title

![CleanShot 2024-08-19 at 09 13 34@2x](https://github.com/user-attachments/assets/c58c68e8-9b71-4f03-9de4-46d498e11c8b)